plugins/modules/wafv2_web_acl.py

```diff
@@ -433,6 +437,18 @@ def create(self, default_action, rules, sampled_requests, cloudwatch_metrics, me
         return self.existing_acl
 
 
+def format_result(result):
+
+    # We were returning details of the Web ACL inside a "web_acl"  parameter on
+    # creation, keep returning it to avoid breaking existing playbooks, but also
+    # return what the docs said we return (and returned when no change happened)
+    retval = dict(result)
+    if "WebACL" in retval:
+        retval.update(retval["WebACL"])
```

Comment:
Does this require an update in the RETURN block?

Reply:
IMO, no.

What's consistently returned, is what's in the return docs.  *Sometimes* web_acl is also returned, however this is for compatibility with what was previously returned.